recipe/meta.yaml

```
@@ -23,11 +24,9 @@ outputs:
         - pip
         - setuptools
         - wheel
-        - packaging >=20.0
-        - typing-extensions
+        - tomli
```

```suggestion
        - tomli  # [py<311]
```
https://github.com/pypa/setuptools_scm/blob/48b1cb547a407b986f674740eae4f54503a141e3/pyproject.toml#L7

---

Thank you! I missed that.